src/sysinfo.cc

```diff
@@ -210,11 +210,11 @@ bool ReadFromFile(std::string const& fname, ArgT* arg) {
   return f.good();
 }
 
-std::pair<bool /*validity*/, bool /*value*/> CpuScalingEnabled(int num_cpus) {
+CPUInfo::Scaling CpuScaling(int num_cpus) {
   // We don't have a valid CPU count, so don't even bother.
-  if (num_cpus <= 0) return std::make_pair(false, false);
+  if (num_cpus <= 0) return CPUInfo::Scaling::UNKNOWN;
 #ifdef BENCHMARK_OS_QNX
-  return std::make_pair(true, false);
+  return CPUInfo::Scaling::DISABLED;
```

Comment:
this should be unknown now i think.

Reply:
So we're doing more of bugfixing also? I just got the previous algo..

Reply:
the PR is about not reporting if we don't get scaling information. if the new contract is to report something useful when we can and not if we can't, then this isn't a bugfix. it's about aligning the code with the new contract you've introduced.